src/src/ESPEasyCore/ESPEasyRules.cpp

```diff
@@ -461,6 +461,27 @@ void parse_string_commands(String& line) {
             && validIntFromString(arg2, endpos)) {
           replacement = arg3.substring(startpos, endpos);
         }
+      #ifndef LIMIT_BUILD_SIZE
+      } else if (cmd_s_lower.equals(F("timetomin")) || cmd_s_lower.equals(F("timetosec"))) {
+        // time to minutes, transform a substring hh:mm to minutes
+        // time to seconds, transform a substring hh:mm:ss to seconds
+        // syntax similar to substring
+        int startpos, endpos = -1;
+        struct tm tm;
+
+        if (validIntFromString(arg1, startpos)
```

Comment:
Why not using existing functions like `timeStringToSeconds` ?
By outputting it to either minutes or seconds depending on the nr of arguments you may introduce unexpected behavior when comparing times.

e.g. `12:34` < `10:00:00`